src/App/App.tsx

```diff
@@ -68,7 +69,28 @@ class App extends Component<IProps, IState> {
 						</section>
 					)
 				}} />
-				<Route path='/saved-playlist' render={() => <PlaylistContainer playlistType={'saved-playlist'} selectedGenre={this.state.selectedGenre} playlists={this.state.playlists} />} />
+				<Route 
+					path='/saved-playlist'
+					render={() => <PlaylistContainer
+						playlistType={'saved-playlist'}
+						selectedGenre={this.state.selectedGenre}
+						playlists={this.state.playlists} 
+					/>} 
+				/>
+				<Route 
+					path='/playlist/:id' 
+					render={({ match }) => {
+						const playlistId: number = +match.params.id;
+						const foundPlaylist: Playlist | undefined = this.state.playlists.find(playlist => {
+							return playlist.id === playlistId
+						});
+						return (
+							<PlaylistDetails
+								playlist={foundPlaylist} 
+							/>
+						);
+					}}
+				/>
```

Comment:
Makes it really easy to see how we are routing them to a more detailed view of the playlist that was generated for them. I wonder, with typescript, if we can do a conditional render to have undefined show a message on the page that there was nothing found for that result and have when a playlist is found be another part of that conditional render if it would still give us the same issue inside of PlaylistDetails.